docs/attributes-registry/process.md

```diff
@@ -11,12 +11,27 @@
 | `process.command` | string | The command used to launch the process (i.e. the command name). On Linux based systems, can be set to the zeroth string in `proc/[pid]/cmdline`. On Windows, can be set to the first parameter extracted from `GetCommandLineW`. | `cmd/otelcol` |
 | `process.command_args` | string[] | All the command arguments (including the command/executable itself) as received by the process. On Linux-based systems (and some other Unixoid systems supporting procfs), can be set according to the list of null-delimited strings extracted from `proc/[pid]/cmdline`. For libc-based executables, this would be the full argv vector passed to `main`. | `[cmd/otecol, --config=config.yaml]` |
 | `process.command_line` | string | The full command used to launch the process as a single string representing the full command. On Windows, can be set to the result of `GetCommandLineW`. Do not set this if you have to assemble it just for monitoring; use `process.command_args` instead. | `C:\cmd\otecol --config="my directory\config.yaml"` |
+| `process.end` | string | The date and time the process ended, in ISO 8601 format. | `2023-11-21T09:26:12.315Z` |
```

Comment:
would it make sense to call it `exit` since it's already used in `exit_code` and more frequently used in relation to processes. Then we can make it a namespace:

`process.exit.time`
`process.exit.code`

We might have more over time
`process.exit.reason`
...

Reply:
Hi @lmolkova, what do you think of keeping `process.end` as is, and leaving the `user` field until the user namespace is available?

I'd like to get everything resolved and move this PR into the Needs Approvals state on the project board soon

Reply:
Exit is a common command that used to terminate the process ([linux](https://www.google.com/search?q=linux+process+exit&rlz=1C1RXQR_enUS1082US1082&oq=linux+process+exit&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQIxgnMgcIAhAAGIAEMggIAxAAGBYYHjIICAQQABgWGB4yCAgFEAAYFhgeMggIBhAAGBYYHjIGCAcQRRg80gEINTQ5OGowajeoAgCwAgA&sourceid=chrome&ie=UTF-8), [windows](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-exitprocess)). 


If we designed these attributes from the scratch, we'd at least entertained the idea of calling them `process.start_time` and `exit_time` since they are too generic without `_time` and since `exit` is much more common term comparing to `end` in the context of processes.

So it seems having `start` and `exit` as namespaces and (`process.start.time`, `process.exit.time` and `process.exit.code`) would provide more clarify and allow to add start/stop info to specific namespaces.

Reply:
I should note that if we go this part, we will not be able to use `start` as a namespace because we already have this field as simple type in ECS and according to our [rules](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#merging-existing-ecs-conventions) we shouldn't change the type (use existing ECS name as a namespace).
With `exit.*` vs `end` we don't have this problem.

In general I am not against changing it to the new structure, but we should be careful with naming

Reply:
I could change to `process.creation` for the namespace instead of start.  Creation is in the [Windows API](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa) and Linux docs (with Linux, the syscalls are fork and exec, not creation specifically;  there are still some places that refer to process creation, like [this](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v1/pids.html?highlight=process+creation))

@lmolkova, @trisch-me, if you agree, I'll change to namespaces for `creation` and `end`.

Reply:
> So there will be then `creation.time`, `exit.time` and `exit.code`?
> 
> I'm ok with this naming. Let's see what others think

Yes, that's what the fields will be renamed to

Reply:
Thank you, sounds good!

Reply:
I've changed the fields to `creation.time`, `exit.time` and `exit.code`